examples/exit-handler-teams.yaml

```diff
@@ -0,0 +1,33 @@
+# This example shows how to hit a teams webhook on an exit handler.
+# In order to configure, create teams webhook url for it.
+# The documentation to do so is here: https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook
+# Replace TEAMS_WEBHOOK_URL_HERE on line 31 with that webhook url
```

Comment:
Small QC: I'm seeing this come through on line 33 of the file, not line 31. Update? 

Smart to call this out, though 👍